internal/testplanet/planet.go

```diff
@@ -167,11 +175,11 @@ func NewWithLogger(log *zap.Logger, satelliteCount, storageNodeCount, uplinkCoun
 
 		node.Dependencies = append(node.Dependencies,
 			closerFunc(func() error {
-				return server.Stop(context.Background())
+				return server.Stop(ctx)
 			}))
 		// bootstrap all the kademlia nodes
 		go func(n *Node) {
-			if err := n.Kademlia.Bootstrap(context.Background()); err != nil {
+			if err := n.Kademlia.Bootstrap(ctx); err != nil {
```

Comment:
make it a `context.TODO()`

Why does it need so much bootstrapping?

Reply:
It was due to a bug with Kademlia StartRefresh that is since fixed.